RFCs/0031-Multi-Test-Run-Finalization.md

```diff
@@ -0,0 +1,137 @@
+# 0031 Multi Test Run Finalization
+
+# Summary
+Introduce APIs to perform data attachments reprocessing (e.g. combining, merging) after two or more independent test executions.
+
+# Motivation
+Today when Test Platform executes tests in parallel only code coverage reports are merged (attachments with uri: `datacollector://microsoft/CodeCoverage/2.0`). For other attachments processing is skipped and all attachments are returned by Test Platform.
+
+The [dotnet test](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test) command is used to execute unit tests in a given solution. The `dotnet test` command builds the solution and runs a test host application for each test project in the solution. However, currently there is no way to combine/merge attachments associated with each project execution. Coverage reports are not merged.
+
+When `Run All Tests` is performed in VS, tests for projects can be executed separately. In this case also combining/merging of data attachments is not performed. Coverage reports are not merged. `Analyze Code Coverage for All Tests` is showing coverage report for only one of the test projects.
```

Comment:
`can be executed separately` -> `can be executed separately based on target platform/target framework amongst other criteria`